spec/factories/activities.rb

```diff
@@ -2,11 +2,11 @@
   factory :activity do
     user
     distance { Faker::Number.decimal(l_digits: 2) }
-    num_drinks { Faker::Number.number(digits: 1) }
-    calories { Faker::Number.number(digits: 3) }
+    num_drinks { (calories / 250).round }
```

Comment:
These changes to the factory add so much better context and scope to our testing, awesome!